daily-patch-information/src/main/java/org/wso2/patchinformation/MainEmailSender.java

```diff
@@ -0,0 +1,132 @@
+//
```

Comment:
The license header should be a block comment, not single line comment. Please correct it in all the places.